lib/ar_lazy_preload/contexts/base_context.rb

```diff
@@ -15,6 +15,7 @@ class BaseContext
       def initialize(records:)
         @records = records.dup
         @records.compact!
+        @records.uniq!
```

Comment:
I've tried to comment this line and specs are still passing. What was the idea?

Reply:
When I run all tests with `$ rake` `integrity_spec.rb` fails with `Rails 6` and `master` branch of Rails. Here is one of the errors output:

```
     Failure/Error: expect(lazy_post_ids).to eq(included_post_ids)

       expected: [[1, 2], [], [], [1, 2], [3, 4], [], [], [3, 4]]
            got: [[1, 1, 2, 2], [], [], [1, 1, 2, 2], [3, 3, 4, 4], [], [], [3, 3, 4, 4]]

       (compared using ==)
     # ./spec/ar_lazy_preload/integrity_spec.rb:100:in `block (3 levels) in <top (required)>'
```